### PR TITLE
UIIN-2604 Ignored hot key command on edit fields.

### PR DESCRIPTION
**Pull request by przemyslawturek**
Ignored hot key command  (ALT + n) on edit fields.

## Purpose
Prevent execute hot key command on edit fields to able input special characters like polish "ń" 

## Approach
By detect target event is edit field 

## Refs
[UIIN-2604](https://issues.folio.org/browse/UIIN-2604)

## Screenshots

Inventory_polish_special_charakter_alt_n

### CODE REVIEW

File: src/components/InstancesList/InstancesList.js [hunk 1]
```diff
@@ -1174,11 +1173,15 @@ class InstancesList extends React.Component {
     const shortcuts = [
       {
         name: 'new',
-        handler: handleKeyCommand(() => {
+        handler: (e) => {
+          const nodeName = e.target?.nodeName || '';
+          const editTextField = (nodeName === 'TEXTAREA') || (nodeName === 'INPUT');
```
Comment (BogdanDenis) [inline src/components/InstancesList/InstancesList.js:1178]: Let's rename it to `isEditTextField` to be consistent with boolean flag naming

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/components/InstancesList/InstancesList.js (hunk 1 above)
Comment (przemyslawturek) [reply src/components/InstancesList/InstancesList.js]: Yeah sure

**Review by BogdanDenis — APPROVED** [review]

**Review by mariia-aloshyna — COMMENTED** [review]
What do you think about passing to `HasCommand` in `InstancesList` component custom `checkScope` func to check whether the active element is not input or textarea? So it will work not only for `n` letter but for others as well

   ```
 const checkScope = () => {
      const ignoreElements = ['TEXTAREA', 'INPUT'];

      return !ignoreElements.includes(document.activeElement.tagName);
    };
```

**Comment by przemyslawturek** [comment]
I think it couse to the same results, but checking scope is cleaner. I'll do the changes. Thanks.

**Review by mariia-aloshyna — APPROVED** [review]

**Pull request merged**